ts/store/reducers/profile.ts

```diff
@@ -29,6 +31,29 @@ const INITIAL_STATE: ProfileState = pot.none;
 export const profileSelector = (state: GlobalState): ProfileState =>
   state.profile;
 
+export const spidEmailSelector = (state: GlobalState) => {
```

Comment:
imho this selector has to return the spid email if it is present, that's it.
In this way if I call the selector and it returns a string, what is the true? is that string a valid email? why I have to check if the string is an email or a simple string (_global.remoteStates.notAvailable_)? 

From an external user the selector should return an option with the email if it is present or none if it isn't

```typescript
export const spidEmailSelector = createSelector(profileSelector, profile => {
  const optionProfile = pot.toOption(profile);
  return optionProfile.map(p => fromNullable(p.spid_email));
});
```

Reply:
I removed the mentioned selector, being no more used. It has been introduced the `profileSpidEmailSelector` instead.